client/lib/products-values/index.js

```diff
@@ -211,6 +213,10 @@ export function isJetpackProduct( product ) {
 	return includes( jetpackProducts, product.product_slug );
 }
 
+export function isJetpackBackup( product ) {
```

Comment:
My personal opinion is that this is not needed right now, but I'm fine to take it as an early adoption of the right logic.